packages/framework/ini-files/config-specs.ini

```diff
@@ -2306,6 +2306,8 @@ opt-set-cmake-var Trilinos_ENABLE_Fortran OFF    BOOL         : OFF
 opt-set-cmake-var Trilinos_ENABLE_COMPLEX_DOUBLE BOOL         : ON
 opt-set-cmake-var CMAKE_CXX_FLAGS                STRING       : -Wall -Wno-clobbered -Wno-vla -Wno-pragmas -Wno-unknown-pragmas -Wno-parentheses -Wno-unused-local-typedefs -Wno-literal-suffix -Wno-deprecated-declarations -Wno-misleading-indentation -Wno-int-in-bool-context -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-nonnull-compare -Wno-address -Wno-inline -Wno-unused-but-set-variable -Wno-unused-variable -Wno-unused-label -Werror -DTRILINOS_HIDE_DEPRECATED_HEADER_WARNINGS
 opt-set-cmake-var TPL_ENABLE_ParMETIS            BOOL FORCE   : OFF
+opt-set-cmake-var TPL_ENABLE_Pnetcdf             BOOL FORCE   : OFF
+opt-set-cmake-var TPL_Netcdf_LIBRARIES           STRING FORCE : -L${NETCDF_C_LIB|ENV};${NETCDF_C_LIB|ENV}/libnetcdf.a;${TPL_HDF5_LIBRARIES|CMAKE}
```

Comment:
And this likely caused the issues with the nompi build.

Reply:
exactly